client/layout/global-sidebar/header.tsx

```diff
@@ -8,8 +9,14 @@ import SidebarNotifications from './menu-items/notifications';
 import { SidebarSearch } from './menu-items/search';
 
 export const GlobalSidebarHeader = () => {
+	const hasEnTranslation = useHasEnTranslation();
 	const translate = useTranslate();
+
+	const isMac = window?.navigator.userAgent && window.navigator.userAgent.indexOf( 'Mac' ) > -1;
+	const searchShortcut = isMac ? '⌘ + K' : 'Ctrl + K';
```

Comment:
I think we have to hide the shortcut on mobile

Reply:
@arthur791004 not sure if I understood what you mean, but the global sidebar search icon is not shown in mobile resolution 🤔 
 
![Screenshot 2024-04-30 at 4 13 42 PM](https://github.com/Automattic/wp-calypso/assets/797888/bc947ab4-b412-4c90-ad5f-73ca64df42a7)

Reply:
Oh yes. How about iPad? But maybe it's fine as people have to tap the icon for a second to see the Tooltip